lib/utility/getTableName.js

```diff
@@ -10,5 +10,6 @@ exports.getTableName = function getTableName (Model) {
   }
   var table = Model.getMeta('table') || parent.name || Model._supermodel || Model.name
   table = table.split('/').pop()
+  table = this.escapeKeys([table])[0]
```

Comment:
actually this leads to a lot of bugs, if you run the integration tests you're going to see it...